### PR TITLE
R9x ad vendor

### PR DESCRIPTION
**Pull request by vishalvanpariya**

### CODE REVIEW

**Review by kristoferbaxter — CHANGES_REQUESTED** [review]
Looks like a formatting error was detected by CI. Can you address @vishalvanpariya?

```
checks.js Running amp lint --local_changes...
[13:39:02] Using task file project/amp.js
[13:39:02] Starting 'lint'...
[13:39:04] 
ads/ads.extern.js
  619:1  error  Delete `⏎`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
  ```

**Comment by vishalvanpariya** [comment]
Hi @kristoferbaxter, errors are fixed now.

**Comment by vishalvanpariya** [comment]
@kristoferbaxter any update on this?

**Comment by vishalvanpariya** [comment]
@kristoferbaxter please merger PR.

File: ads/vendors/andbeyond.md [hunk 1]
```diff
@@ -5,8 +5,8 @@
 ```html
 <amp-ad width="300" height="250"
     type="andbeyond"
-    data-siteid="40827"
-    data-slotpath="/103512698/AMP_COMPONENT_TEST_1"
+    data-siteid="9999"
```
Comment (powerivq) [inline ads/vendors/andbeyond.md:8]: That is another vendor's document. Can you add yours instead?

**Review by powerivq — CHANGES_REQUESTED** [review]

**Comment by powerivq** [comment]
We are sweeping through PRs and found this. If you are still interested, please sign the CLA and fix the errors. @vishalvanpariya

**Pull request closed without merging**